test/uk/gov/hmrc/gform/sharedmodel/formtemplate/generators/FormTemplateGen.scala

```diff
@@ -51,6 +51,8 @@ trait FormTemplateGen {
       templateName <- templateNameGen
     } yield WebChat(ChatRoomId(roomId), templateName)
 
+  def toLocalisedString(s: String): LocalisedString = LocalisedString(Map(LangADT.En -> s))
```

Comment:
This function already exists, let's reuse existing one.

Reply:
swapped it use the helpers toLocalisedString